contributors.txt

```diff
@@ -1,5 +1,7 @@
-﻿This is a list of people and organizations that contributed to the NVDA project in various ways since the beginning of the project. For an overview of code contributors, see also
-https://github.com/nvaccess/nvda/graphs/contributors
+﻿This is a list of people and organizations that contributed to the NVDA project in various ways since the beginning of the project.
+For an overview of code contributors, see also https://github.com/nvaccess/nvda/graphs/contributors.
+For a list of active contributors, see the experts list https://github.com/nvaccess/nvda/blob/master/projectDocs/community/expertsList.md
+This list is now archived.
```

Comment:
Please move this sentence just after the first sentence of the file for clarity.